ee/vellum_ee/workflows/display/nodes/vellum/conditional_node.py

```diff
@@ -44,13 +44,15 @@ class RuleIdMap:
 class BaseConditionalNodeDisplay(BaseNodeVellumDisplay[_ConditionalNodeType], Generic[_ConditionalNodeType]):
     source_handle_ids: ClassVar[Dict[int, UUID]]
     rule_ids: ClassVar[List[RuleIdMap]]
+    condition_ids: ClassVar[Dict[int, UUID]]
 
     def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> JsonObject:
         node = self._node
         node_id = self.node_id
 
         node_inputs: List[NodeInput] = []
         source_handle_ids = self._get_source_handle_ids() or {}
+        condition_ids = self._get_condition_ids() or {}
```

Comment:
Assuming we go with my suggestion above, we should raise an exception if the user explicitly specified condition_ids on the class, but the length of the array they provided doesn't match the number of conditions